android/app/lombok.config

```diff
@@ -0,0 +1,2 @@
+lombok.anyConstructor.suppressConstructorProperties = true
+lombok.addGeneratedAnnotation = false
```

Comment:
@iamareebjamal I don't think it will create any problem. We have declared property according to jsonapi and kebabspace. 

Setting `lombok.getter.noIsPrefix = true` will just rename getters and setters for boolean type.
if true then it uses get prefix and if false is
example,
`isBookmarked` is property of Session model. If `noIsPrefix` is `true` then getter for it will be `getIsBookmarked` else `isBookmarked`

Please tell which problem you trying to say.